charts/atlantis/stages/production-terraform-cloud/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Use this values file after the initial GitHub app registration. This file needs the ATLANTIS_GH_APP_ID generated by the initial setup process.
+# See initial_setup_values.yaml for the initial setup.
+
+serviceAccount:
+  name: atlantis
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::749942219478:role/atlantis-ops-role
+
+environmentSecrets:
+  - name: ATLANTIS_GH_WEBHOOK_SECRET
+    secretKeyRef:
+      name: atlantis-secret
+      key: github_secret
+
+environment:
+  # Set the ATLANTIS_GH_APP_ID to the id generated by the initial app setup process.
+  ATLANTIS_GH_APP_ID: 114426
```

Comment:
Not sure if we can use the same Github App ID or if we should make a new one.

Reply:
Unless process is easy to create a new APPI_ID and setup integration. We do have admin access to few repos but not all. Apart from Audit any other advantage of using separate APP_ID?

Reply:
Got it. I'll go ahead and create a ticket with IT.

@rarkebauer for the pem file and secret of the new app, I think I can just use SSM and a create an `ExternalSecret` similar to what we did for [Terraform Cloud API Key](https://github.com/procore/atlantis-helm-chart/blob/main/charts/atlantis/templates/secret-tfcloud.yaml).

Reply:
Yes, this is what we did with the original atlantis secrets but I think its preferable to use Externalsecrets over terraforming kube secrets: https://github.com/procore/terraform-aws-apps/blob/ace026e203220e086161634a9eae5042ad20b9ea/modules/atlantis/secrets.tf#L1
You'll have to do the IAM in terraform still https://github.com/procore/terraform-aws-apps/blob/master/modules/atlantis/iam.tf